doc/LICENSE

```diff
@@ -2,11 +2,14 @@ The Coq Reference Manual is a collective work from the Coq Development
 Team whose members are listed in the file CREDITS of the Coq source
 package. All related documents (the LaTeX and BibTeX sources, the
 embedded png files, and the PostScript, PDF and html outputs) are
-copyright (c) INRIA 1999-2006. The material connected to the Reference
-Manual may be distributed only subject to the terms and conditions set
-forth in the Open Publication License, v1.0 or later (the latest
-version is presently available at http://www.opencontent.org/openpub/).
-Options A and B are *not* elected.
+copyright (c) INRIA 1999-2006, with the exception of the Ubuntu font files
+(UbuntuMono-Square.ttf and UbuntuMono-B.ttf) which are Copyright 2010,2011
```

Comment:
UbuntuMono-Square.ttf is actually generated from UbuntuMono-Regular.

Reply:
So should I update the text? If yes, what should I put/remove?

Reply:
Unclear; I just meant to point it out.  Maybe "derived from file Copyright …"